theory/energetic sources/supernovae.md

```diff
@@ -7,3 +7,22 @@ nav_order: 4
 ---
 
 ## Supernovae
+
+A supernova is a powerful explosion that marks the final stage of a star's life. As the star approaches its death, its core begins to collapse under immense pressure. This collapse forces protons and electrons to combine, forming neutrons. The star's outer layers are then expelled into space in a spectacular explosion.
+
+Supernovae release huge amounts of energy. The amount of energy released can go up to 10^51 ergs.
```

Comment:
Use `$$10^{51}$$` for mathjax compatibility

Reply:
fixed it